docs/tutorials/Bonding_and_Charge/ELF.md

```diff
@@ -0,0 +1,144 @@
+
+What you should immediately see is something like this :
+
+![HF initial vesta output](HF_init.png)
+
+You'll be able to see it more properly by rotating it around yourself, but the yellow represents the electron cloud, and there are 2 atoms in the middle (naturally the small one is hydrogen and large one is fluorine) which are surrounded by said cloud.
+
+We can already see evidence of lone pairs/hydrogen bonding : the 'circles' of yellow outside the big cloud even visually resemble lone pairs, while chunks going even further away could potentially hint at them trying to form hydrogen bonds with adjacent cells.
+
+We can visualise this better by clicking `Utilities -> 2D Data Display` in Vesta. That'll open up a new window - in it, click `Slice...` and select the miller indices to be 001. Apply that and you should get an image like this :
+
+![ELF electron distribution](HF_slice.png){width="30%"}
+
+This gives us an interesting picture: the small regions of high electron density look even more like what we'd expect of lone pairs. The nitrogen has 3 around it (as expected), yet the hydrogen also has 1 - this is not what we'd expect. However, considering that this will be interacting with adjacent cells, this is a good indication that it has formed a hydrogen bond.
+
+## Water and ammonia
+
+The procedure for H~2~O and NH~3~ is almost identical. We use exactly the same `param` file as above, and perform the same procedures following the `elf_fmt` output. The `cell` files we use are:
+
+*H2O.cell*
+```
+%block lattice_abc
+5 5 5
+90 90 90
+%endblock lattice_abc
+
+%block positions_abs
+O 2.5 2.5 2.5
+H 2.5 1.95 3.29
+H 2.5 1.95 1.72
+%endblock positions_abs
+```
+
+for water and
+
+*NH3.cell*
+```
+%BLOCK LATTICE_ABC
+8 8 8
+90 90 90
+%ENDBLOCK LATTICE_ABC
+
+%BLOCK POSITIONS_ABS
+N 4 4 4
+H 4 4 5
+H 4.95 4 3.67
+H 3.53 3.18 3.67
+%ENDBLOCK POSITIONS_ABS
+
+
+```
+
+For ammonia.
+
+From then, we again go to `2D display data` and `Slice...`. This is where the procedure might be a bit different: by shift clicking on all the atoms in H~2~O (and the atoms within the plane you wish to look at in NH~3~) and clicking `Calculate the best plane for the selected atoms` it'll give us the plane that we may be most interested in. In the case of water it is a simple (100), but for NH~3~ you may wish to play around with it by selecting different combinations - alternatively manually choose a plane in which you are interested
+
+!!! note
+    In VESTA, plane selection may work a bit differently from how you'd expect. In this example, since you're cutting through the middle you'd normally call it a (200) plane. However, here it is considered a (100) plane 2.5$\mathring{\text{A}}$ (or 0.5d - since the unit cell lengths are 5$\mathring{\text{A}}$) from the origin
```

Comment:
This sentence seems half finished for some reason. Perhaps because it doesn't say why it's different. Also missing full stop.